data/human/ships.txt

```diff
@@ -1373,7 +1373,7 @@ ship "Cutthroat"
 		"Twin Modified Blaster" 2
 		"Torpedo Pod" 2
 		"Torpedo Storage Rack"
-		"Torpedo" 21
+		"Torpedo" 23
```

Comment:
You haven't changed the capacity of torpedo pods, so this doesn't need to be changed.